### PR TITLE
[validationAddPostPanel] Validation add post panel

### PR DESCRIPTION
**Pull request by arowinsky**
-added validation into form to create new post

### CODE REVIEW

**Comment by Maciek246** [comment]
In commits: [branch_name]. Dont write inside random names.

File: src/App.js [hunk 1]
```diff
@@ -18,8 +20,9 @@ class App extends Component {
             <MainNavbar />
             <Route exact path="/" component={MainPage} />
             <Route exact path="/user/:username" component={UserPanel} />
-            <Route exact path="/:slug" component={BlogLovWeb} />
+            <Route exact path="/:addblog/slug" component={BlogLovWeb} />
```
Comment (Maciek246) [inline src/App.js]: Why? Its nonsense. We don't have variable 'addblog'. Nowhere. In this form routing our app is broken.

File: src/App.js [hunk 2]
```diff
@@ -18,8 +20,9 @@ class App extends Component {
...
             <Route exact path="/:slug/addpost" component={AddPost} />
+            <Route exact path="/:addblog" component={AddBlog} />
```
Comment (Maciek246) [inline src/App.js]: Again. ":" char is for 'variable' in url. 'addblog' is static route. Remove ":".

File: src/components/AddBlogPanel/index.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+class AddBlogPanel extends Component {
+    constructor() {
+        super();
+        this.state = {
+            fields: {},
+            errors: {}
+        }
+
+        this.handleChange = this.handleChange.bind(this);
+        this.submitaddBlogForm = this.submitaddBlogForm.bind(this);
```
Comment (Maciek246) [inline src/components/AddBlogPanel/index.js]: In ES6 exists arrow function which gives a possibility avoid a bind(this). But it's not a necessary.

File: src/components/AddBlogPanel/index.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    handleChange(e) {
+        let fields = this.state.fields;
+        fields[e.target.name] = e.target.value;
+        this.setState({
+            fields
+        });
+
+    }
+
+    submitaddBlogForm(e) {
```
Comment (Maciek246) [inline src/components/AddBlogPanel/index.js]: Use camelcase

File: src/components/MainNavbar/index.js [hunk 5]
```diff
@@ -35,6 +35,7 @@ class MainNavbar extends Component {
             {isAuthenticated ?
               <div className="main-navbar main-navbar--content-right">
                 <Link to="/user/:username"><button className="button--main-navbar blue">Twoje konto</button></Link>
+                <Link to="/:addblog"><button className="button--main-navbar blue">Utwórz nowy blog</button></Link>
```
Comment (Maciek246) [inline src/components/MainNavbar/index.js]: Ditto: 'addblog' is static route not route with a variable.

File: src/components/AddPostPanel/index.js [hunk 6]
```diff
@@ -1,27 +1,80 @@
...
+class AddPostPanel extends Component {
+    constructor() {
+        super();
+        this.state = {
+            fields: {},
+            errors: {}
+        }
 
+        this.handleChange = this.handleChange.bind(this);
+        this.submitaddPostForm = this.submitaddPostForm.bind(this);
```
Comment (Maciek246) [inline src/components/AddPostPanel/index.js]: Ditto: ES6 syntax

File: src/components/AddPostPanel/index.js [hunk 7]
```diff
@@ -1,27 +1,80 @@
...
+    handleChange(e) {
+        let fields = this.state.fields;
+        fields[e.target.name] = e.target.value;
+        this.setState({
+            fields
+        });
 
+    }
+
+    submitaddPostForm(e) {
```
Comment (Maciek246) [inline src/components/AddPostPanel/index.js]: Ditto: camelcase

**Review by Maciek246 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (Maciek246) [inline src/App.js]: This isn't fixed.

File: src/App.js (hunk 2 above)
Comment (Maciek246) [inline src/App.js]: Ditto: In this case, you sold remove only ":" from this line.

**Review by Maciek246 — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (arowinsky) [reply src/App.js]: I wrote to you that when it change this is another mistake and you wrote to me that you will change it

File: src/App.js (hunk 2 above)
Comment (arowinsky) [reply src/App.js]: I wrote to you that when it change this is another mistake and you wrote to me that you will change it

File: src/App.js (hunk 1 above)
Comment (Maciek246) [reply src/App.js]: Yes, but you should comment these lines and write a comment with info about it.

**Review by Maciek246 — APPROVED** [review]

**Pull request merged**